x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -21,48 +21,27 @@ const environmentLabels: Record<string, string> = {
   ),
 };
 
+export function getEnvironmentLabel(environment: string) {
+  return environmentLabels[environment] || environment;
+}
+
 export const ENVIRONMENT_ALL = {
-  esFieldValue: undefined,
   value: ENVIRONMENT_ALL_VALUE,
-  text: environmentLabels[ENVIRONMENT_ALL_VALUE],
+  text: getEnvironmentLabel(ENVIRONMENT_ALL_VALUE),
 };
 
 export const ENVIRONMENT_NOT_DEFINED = {
-  esFieldValue: undefined,
   value: ENVIRONMENT_NOT_DEFINED_VALUE,
-  text: environmentLabels[ENVIRONMENT_NOT_DEFINED_VALUE],
+  text: getEnvironmentLabel(ENVIRONMENT_NOT_DEFINED_VALUE),
 };
 
-export function getEnvironmentLabel(environment: string) {
-  return environmentLabels[environment] || environment;
-}
-
-export function omitEsFieldValue({
-  esFieldValue,
-  value,
-  text,
-}: {
-  esFieldValue?: string;
-  value: string;
-  text: string;
-}) {
-  return { value, text };
-}
-
 export function parseEnvironmentUrlParam(environment: string) {
```

Comment:
I don't think this name is correct anymore. I think what it's doing is just converting a url param to an es field value. Should we rename it to something that captures that?

Reply:
Renaming it to `getEnvironmentEsFieldValue`